Sources/SnowManLogicSpecs/ViewSpec.swift

```diff
@@ -0,0 +1,106 @@
+                    expect(view.receiveGuess(io.myReadLine)).to.equal("hello")
+           		}
+
+           		it("should return the user input if it is a single letter") {
+                		io.stubbedUserInput = ["a"]
+                		expect(view.receiveGuess(io.myReadLine)).to.equal("a")
+            		}
+
+            		it("should return an error message if the user input is a number") {   
+                		io.stubbedUserInput = ["1", "fail"]
+                		expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+            		}
+
+                	it("should return an error message if the user input is a non-letter character") {
+                        	io.stubbedUserInput = ["?", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user input is a combination of valid and invalid inputs") {
+                        	io.stubbedUserInput = ["?a", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+
+                    	it("should return an error message if the user hits return") {
+                        	io.stubbedUserInput = ["", "fail"]
+                        	expect(view.receiveGuess(io.myReadLine)).to.equal("fail")
+                    	}
+        	}
+
+        	describe("#assignBlanks") {
+            		var gameWord: String!
+            		var correctGuesses: [String]!
+            
+            		before() {
+                		gameWord = "hello"
+                		correctGuesses = []
+            		}
+            
+            		it("should return only blanks for unguessed word") {
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("__  __  __  __  __ ")
+            		}
+
+            		it("should return no blanks if the word has been guessed") {
+                		correctGuesses = ["h", "e", "l", "o"]
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("h e l l o")
+            		}
+
+            		it("should return blanks and letters for partially guessed word") {
+                		correctGuesses = ["h", "e"]
+                		expect(view.assignBlanks(gameWord, correctGuesses: correctGuesses)).to.equal("h e __  __  __ ")
+            		}
+
+        	}
+
+	        describe("#displayRemainingGuesses") {
+        		it("should return the number of guesses remaining") {
+                    		var remainingGuesses = 4
+                    		expect(view.displayRemainingGuesses(4)).to.equal("You have 4 remaining guesses")
+                
+                    		remainingGuesses = 0
+                    		expect(view.displayRemainingGuesses(0)).to.equal("You have 0 remaining guesses")
+
+                    		remainingGuesses = -2
+                    		expect(view.displayRemainingGuesses(-2)).to.equal("You have -2 remaining guesses")
```

Comment:
Cool. Can remainingGuesses ever be negative?  If so, is there a test to indicate a standard response of `"You have 0 remaining guesses"`